openapi/paths/credit-memos@{id}@void.yaml

```
@@ -0,0 +1,25 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+post:
+  x-products:
+    - Core
+  tags:
+    - Credit memos
+  summary: Void a credit memo
+  operationId: PostCreditMemoVoid
+  x-sdk-operation-name: void
+  description: |
+    Void a credit memo with specified identifier string.
+  responses:
+    '201':
+      description: Credit memo was voided successfully.
```

nitpick: avoid past tense (where possible)
```suggestion
      description: Credit memo successfully voided .
```